cmd/srcd/cmd/utils.go

```diff
@@ -0,0 +1,100 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Defered prints message only after specified timeout
+// it supports printing message with spinner and displaying logs in case timeout expired
+// use InputFn attribute to provide such logs
```

Comment:
maybe it makes sense to mention incompatibility of spinner & logs.

Reply:
Made the comment clearer.